torch/csrc/autograd/python_function.cpp

```diff
@@ -663,10 +663,22 @@ PyObject* THPFunction_name(PyObject *self, PyObject* noargs) {
 PyObject *THPFunction_apply(PyObject *cls, PyObject *inputs)
 {
   HANDLE_TH_ERRORS
-  RECORD_FUNCTION(
-    ((PyTypeObject*)cls)->tp_name,
-    std::vector<c10::IValue>(),
-    at::sequence_number::peek());
+  auto info_pair = unpack_input<false>(inputs);
+  auto iv_inputs = std::vector<c10::IValue>();
```

Comment:
Can this definition be moved inside the if statement?

Reply:
Done, thx!